renderer/main.js

```diff
@@ -1213,11 +1213,7 @@ function openTorrentContextMenu (infoHash) {
 }
 
 function getTorrentPath (torrentSummary) {
-  var itemPath = path.join(torrentSummary.path, torrentSummary.files[0].path)
-  if (torrentSummary.files.length > 1) {
-    itemPath = path.dirname(itemPath)
-  }
-  return itemPath
+  return path.join(torrentSummary.path, torrentSummary.name)
```

Comment:
Is this the bug?

1. create a folder that contains a single file
2. drop the folder onto WTD
3. right click -> "remove data file"
4. File is deleted, but folders remain

This is unfortunately not easily fixable. The torrent spec for single file torrents doesn't allow specifying a path. See [the spec here](https://wiki.theory.org/BitTorrentSpecification#Info_in_Single_File_Mode). If it's a single file torrent, you can just have a name, no path.

We could fix by storing the original folder the user dropped, and deleting that when they delete a single file torrent, but this isn't ideal because it's complicated and other files might have been added to the folder in the meantime.